config/common/temporal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+use Temporal\DataConverter\DataConverter;
+use Temporal\DataConverter\DataConverterInterface;
+use Temporal\Worker\Transport\Goridge;
+use Temporal\Worker\Transport\RPCConnectionInterface;
+use Temporal\Worker\WorkerOptions;
+use Temporal\WorkerFactory;
+
+/**
+ * @var $params array
+ */
+
+if (!($params['yiisoft/yii-runner-roadrunner']['temporal']['enabled'] ?? false)) {
+    return [];
+}
+
+$options = $params['yiisoft/yii-runner-roadrunner']['temporal']['options'] ?? [];
+
+
+return [
+    'tag@temporal.workflow' => [],
```

Comment:
What is it for?

Reply:
You need to mark your workflows and activities with these tags

Reply:
It should be in the readme then, not here.

Reply:
It must be here to make temporal works at least. But I'll add it to the readme